### ISSUE TITLE
Live updating graph?

### ISSUE THREAD

**Issue by drewsed** [daddycocoaman/BeaconGraph#6]
Hi, 
me again ;)

Do you know about [Kismet](https://www.kismetwireless.net/) and the Kismet [Python parser](https://pypi.org/project/kismet/)?

I think it wold be nice, to populate the database in "realtime" and change the color of the relationship arrows after a client was "not seen" for lets say 15 to 30 minutes. In this case we could get a better overview on the surounding wifi stations and their clients.

What do you think?

Cheers,
drewsed

**Comment by daddycocoaman** [comment]
I don't think it's a terrible idea but it sounds like a really long term goal. The Python library you linked to is the incorrect one, but I'm not entirely sure how up to date the existing one is. [Python-Kismet-Rest](https://github.com/kismetwireless/python-kismet-rest) Kismet added an [eventbus websocket](https://www.kismetwireless.net/docs/devel/webui_rest/eventbus/) that is probably useful though. 

I don't particularly have a roadmap for BeaconGraph since wireless assessments aren't a part of my day job or hobbies but I think this can eventually be implemented.

**Comment by drewsed** [comment]
You are right, i linked the wrong python lib, but fortunately you found the right one ;)

A year ago I did some research but i am also not very into WiFi things, thought you're interested in this so i posted this idea for completeness sake.

Maybe it is helpful for someone else who wants to contribute to your project.

Anyway, thanks for your answer!
